.goreleaser.yaml

```diff
@@ -0,0 +1,126 @@
+    mod_timestamp: "{{ .CommitTimestamp }}"
+
+
+archives:
+  - format: binary
+    name_template: >-
+      {{ .ProjectName }}-{{ .Os }}
+  - id: archive
+    format: tar.gz
+    name_template: >-
+      {{ .ProjectName }}-{{ .Os }}
+    # use zip for windows archives
+    format_overrides:
+      - goos: windows
+        format: zip
+
+dockers:
+  - dockerfile: image/Dockerfile
+    ids:
+      - kube-linter
+    image_templates:
+      - "ghcr.io/stackrox/kube-linter:latest"
+      - "ghcr.io/stackrox/kube-linter:{{ .Tag }}"
+      - "stackrox/kube-linter:latest"
+      - "stackrox/kube-linter:{{ .Tag }}"
+    build_flag_templates:
+      - "--pull"
+      - "--platform=linux/amd64"
+      - "--label=org.opencontainers.image.title={{ .ProjectName }}"
+      - "--label=org.opencontainers.image.vendor=RedHat"
+      - "--label=org.opencontainers.image.description=Kube-Linter {{ .Version }}"
+      - "--label=org.opencontainers.image.url=https://kubelinter.io"
+      - "--label=org.opencontainers.image.documentation=https://docs.kubelinter.io/"
+      - "--label=org.opencontainers.image.source={{ .GitURL }}"
+      - "--label=org.opencontainers.image.licenses=Apache"
+      - "--label=org.opencontainers.image.version={{ .Version }}"
+      - "--label=org.opencontainers.image.revision={{ .FullCommit }}"
+      - "--label=org.opencontainers.image.created={{ .CommitDate }}"
+
+  - dockerfile: image/Dockerfile_alpine
+    ids:
+      - kube-linter
+    image_templates:
+      - "ghcr.io/stackrox/kube-linter:latest-alpine"
+      - "ghcr.io/stackrox/kube-linter:{{ .Tag }}-alpine"
+      - "stackrox/kube-linter:latest-alpine"
+      - "stackrox/kube-linter:{{ .Tag }}-alpine"
+    build_flag_templates:
+      - "--pull"
+      - "--platform=linux/amd64"
+      - "--label=org.opencontainers.image.title={{ .ProjectName }}"
+      - "--label=org.opencontainers.image.vendor=RedHat"
+      - "--label=org.opencontainers.image.description=Kube-Linter {{ .Version }}"
+      - "--label=org.opencontainers.image.url=https://kubelinter.io"
+      - "--label=org.opencontainers.image.documentation=https://docs.kubelinter.io/"
+      - "--label=org.opencontainers.image.source={{ .GitURL }}"
+      - "--label=org.opencontainers.image.licenses=Apache"
+      - "--label=org.opencontainers.image.version={{ .Version }}"
+      - "--label=org.opencontainers.image.revision={{ .FullCommit }}"
+      - "--label=org.opencontainers.image.created={{ .CommitDate }}"
+
+docker_manifests:
+  - name_template: "ghcr.io/janisz/kube-linter:{{ .Tag }}"
+    image_templates:
+      - "ghcr.io/janisz/kube-linter:{{ .Tag }}"
+
+checksum:
+  disable: true
+
+signs:
+  - cmd: cosign
+    stdin: "{{ .Env.COSIGN_PASSWORD }}"
+    args:
+      - "sign-blob"
+      - "--key"
+      - "env://KEY"
+      - "--output-signature=${signature}"
+      - "${artifact}"
+      - "--yes"
+    artifacts: all
+
+docker_signs:
+  - cmd: cosign
+    env:
+      - COSIGN_EXPERIMENTAL=1
```

Comment:
We should be able to drop the `COSIGN_EXPERIMENTAL` with [Cosign 2.0](https://blog.sigstore.dev/cosign-2-0-released/).
```suggestion

```